jekyll/.ruby-version

```diff
@@ -1 +0,0 @@
-../.ruby-version
```

Comment:
We need to stay 3.3.3 for now because Netlify uses rvm, which Ruby 3.3.4+ currently fails to build on its
containers due to:

- https://github.com/ruby/bigdecimal/issues/297
- https://github.com/rubygems/rubygems/issues/8015

And because Netlify doesn't allow installing apt packages, we can't address the issue to make the install succeed. So instead, we need to use a version of Ruby that can be built on Netlify's containers.